lib/offline/storage.js

```diff
@@ -288,23 +288,29 @@ shaka.offline.Storage.prototype.remove = function(content) {
   return this.initIfNeeded_().then(function() {
     this.checkDestroyed_();
     return this.dbEngine_.get('manifest', manifestId);
-  }.bind(this)).then(function(/** ?shakaExtern.ManifestDB */ data) {
-    this.checkDestroyed_();
-    if (!data) {
-      throw new shaka.util.Error(
-          shaka.util.Error.Category.STORAGE,
-          shaka.util.Error.Code.REQUESTED_ITEM_NOT_FOUND, uri);
-    }
-    manifestDb = data;
-    var manifest =
-        shaka.offline.OfflineManifestParser.reconstructManifest(manifestDb);
-    var netEngine = this.player_.getNetworkingEngine();
-    goog.asserts.assert(netEngine, 'Player must not be destroyed');
-    drmEngine =
-        new shaka.media.DrmEngine(netEngine, onError, function() {});
-    drmEngine.configure(this.player_.getConfiguration().drm);
-    return drmEngine.init(manifest, true /* isOffline */);
-  }.bind(this)).then(function() {
+  }.bind(this)).then((
+      /**
+       * @param {?shakaExtern.ManifestDB} data
+       * @return {!Promise}
+       */
+      function(data) {
+        this.checkDestroyed_();
+        if (!data) {
+          throw new shaka.util.Error(
+              shaka.util.Error.Category.STORAGE,
+              shaka.util.Error.Code.REQUESTED_ITEM_NOT_FOUND, uri);
+        }
+        manifestDb = data;
+        var manifest = shaka.offline.OfflineManifestParser.
+                reconstructManifest(manifestDb);
```

Comment:
Style nit: indentation looks off here.  Should be +4 spaces from "var manifest", if I'm not mistaken.